src/controllers/accommodation.js

```diff
@@ -163,5 +163,33 @@ export default class accomodationFacility {
       return res.status(500).json({ status: 500, errorMessage: error });
     }
   }
+
+  static likeAccommodation(req, res) {
+    const accommId = req.params.id;
+    Accommodation
+      .findOne({ where: { id: accommId } }).then((accomm) => {
```

Comment:
What will happen if it doesn't find an acommodation or it throws an error?

Reply:
There is a middleware checks for that

Reply:
<img width="757" alt="Screen Shot 2020-03-21 at 21 51 03" src="https://user-images.githubusercontent.com/36619897/77235412-1a742900-6bbe-11ea-8870-fbc521abe0d0.png">

this a middleware that checks everything for that function

Reply:
I have managed to solve these problems